vernac/declaremods.mli

```diff
@@ -129,3 +128,6 @@ type module_objects = private
   }
 
 val modmap : unit -> module_objects Names.MPmap.t
+
+(** This likely wants to be moved*)
+val data_of_path : Names.ModPath.t -> Names.Label.t -> Libobject.Data.t option
```

Comment:
This API seems very wrong to me. If I have eg
~~~coq
Definition foo := 0.

Ltac2 foo () := ().
~~~
and I ask `data_of_path current_mp "foo"`, there is no sensible result.

Reply:
Indeed this only works for loaded modules, do you want this API to query `Lib.contents` too?

(`iter_all_segments` used to do that)

Reply:
This has nothing to do with loaded module, it's about using the same name in different namespaces.

Reply:
Oh yes, let me look into that, thanks for the clarification.

Indeed the document manager for example when drawing a module outline, want to consume a list of all the objects in a module with a kind and a location (and maybe the doc) , so indeed I was thinking that I could get rid of the dumpglob table and put that information here.

I see several options, let me experiment a bit first.